packages/expo-cli/src/commands/build-native/UrlUtils.ts

```diff
@@ -0,0 +1,25 @@
+export function getExpoDomainUrl() {
```

Comment:
this logic should be in xdl and internal to TurtleApi class
All that is using Turtle should specify the only path like `/build/status`.